### DISCUSSION TITLE
Other Books (non Audible)

### DISCUSSION THREAD

**Discussion by Oleo75** [openaudible/openaudible#748] [Q&A] [answered]
I have some older CD books on tape that I would like to backup so I can listed in my car that no-longer has a CD player.  Will this program so that?  Any DRM concerns?

**Comment by openaudible** [comment, marked as answer]
No, it's a bit of a pain and we don't do much in the way of CD -> MP3/M4B.
So, step one: Burn each cd to MP3 or M4Bs, give each a name.
Continue for each CD, and make sure all Mp3's/M4Bs are in the right order alphabetically.
Then use OA to Join Audio Files... The new book will be added to your library as a single book.
Test and delete old individual CD files.
